src/main/java/command/DeleteCommand.java

```diff
@@ -0,0 +1,61 @@
+package command;
+
+import java.io.IOException;
+
+import exception.NoIndexException;
+import parser.Parser;
+import storage.Storage;
+import task.Task;
+import task.TaskList;
+import ui.Ui;
+
+
+
+/**
+ * DeleteCommand would execute the program when user specify
+ * "delete" as the command. This would automatically delete the task
+ * specified by user in number via TaskList, show message of the deleted
+ * task via Ui, and update the external file.
+ */
+public class DeleteCommand extends Command {
+
+    private String command;
+
+
+    /**
+     * Constructs a DeleteCommand with the given
+     * user command.
+     *
+     * @param command String user command
+     */
+    public DeleteCommand(String command) {
+        super();
+        this.command = command;
+    }
+
+    /**
+     * Executes the parsed user command. The result is:
+     * 1.Removes the tasks based on the index number specified
+     * in the user command in the task list
```

Comment:
It seems that punctuation is missing here. I noticed that are also similar problems in several other places.